v2/internal/controllers/crd_dataprotection_backupinstance_20231101_test.go

```diff
@@ -32,7 +32,7 @@ func Test_DataProtection_BackupInstance_20231101_CRUD(t *testing.T) {
 	// Create a test resource group and wait until the operation is completed, where the globalTestContext is a global object that provides the necessary context and utilities for testing.
 	tc := globalTestContext.ForTest(t)
 	rg := tc.CreateTestResourceGroupAndWait()
-	contributorRoleId := fmt.Sprintf("/subscriptions/%s/providers/Microsoft.Authorization/roleDefinitions/17d1049b-9a84-46fb-8f53-869881c3d3ab", tc.AzureSubscription)
+	contributorRoleId := fmt.Sprintf("/subscriptions/%s/providers/Microsoft.Authorization/roleDefinitions/b24988ac-6180-42a0-ab88-20f7382dd24c", tc.AzureSubscription)
```

Comment:
How did the contributor role change...? It's always the same GUID isn't it?

Is this new GUID a different role?

Reply:
`17d1049b-9a84-46fb-8f53-869881c3d3ab` is storageAccountContributor and this one requires just `Contributor` to ResourceGroup. 

Not sure if it was incorrect or RP changed something but this one works now.